samples/nrf9160/aws_iot/src/main.c

```diff
@@ -493,15 +501,30 @@ void main(void)
 {
 	int err;
 
-	LOG_INF("The AWS IoT sample started, version: %s", CONFIG_APP_VERSION);
+	LOG_INF("The AWS IoT sample started, version: %s", CONFIG_AWS_IOT_SAMPLE_APP_VERSION);
 
 	cJSON_Init();
 
 #if defined(CONFIG_NRF_MODEM_LIB)
 	nrf_modem_lib_dfu_handler();
 #endif
 
+#if IS_ENABLED(CONFIG_AWS_IOT_SAMPLE_DEVICE_ID_USE_HW_ID)
+	static char device_id[HW_ID_LEN];
+
+	err = hw_id_get(device_id, HW_ID_LEN);
+	if (err) {
+		LOG_ERR("Failed to retrieve device ID, error: %d", err);
+	}
+	static struct aws_iot_config config;
+
+	config.client_id = device_id;
+	config.client_id_len = HW_ID_LEN;
+	LOG_INF("Using UUID as device id: %s", device_id);
```

Comment:
This will not be a UUID by default, as it's selected by option `HW_ID_LIBRARY_SOURCE`. Suggest changing to just `Device id: %s`